### DISCUSSION TITLE
Donation

### DISCUSSION THREAD

**Discussion by daniel-sousa-me** [CarlosDerSeher/snapclient#32] [General]
I just got this working! A way cheaper solution than a full raspberry pi lol

Can I donate a few bucks to you as a thank you? How do I do it?

**Comment by CarlosDerSeher** [comment]
Hahaha I have never thought about taking donations :) no idea how to do it, PayPal maybe, but there will be fees I guess. Also I have to check if there are any tax issues coming for me if I do take donations. As soon as I am clear about these things I will tell you.

Another question, which Hardware platform do you use for your clients?

**Reply by CarlosDerSeher** [reply]
@daniel-sousa-me Ok now I gave this some thought and came to a conclusion. If you really want to donate something I find it best if you gave the money to these guys

https://spenden.epicenter.works/

They will do something really useful with it.

Sadly the page is German only, so if you need help just give me a word. Maybe google translate can be useful too.

Thank you :)

**Reply by daniel-sousa-me** [reply]
That's a great plan! Consider it done!
